### ISSUE TITLE
disabling and enabling channels

### ISSUE THREAD

**Issue by steff123** [accumulator/charge-lnd#31]
With lnd 0.13 it is possible to disable channels 
lncli updatechanstatus --funding_txid 123 --output_index 0 --action disable

It would be helpfull if one can disable and enable a channel then with charge-lnd too, e.g. when the outbound amount is very low. At the moment I raise fees in that case. But it has the sideeffect that the average fee of the node on amboss is raising too.

**Comment by accumulator** [comment]
There are two problems with disabling channels.

The first is that when disabling a channel, there's also no way the channel can get back to being more balanced as all forwarding is stopped.

The second is that after disabling a channel, how to decide when to enable it again. 
The only way I can think of right now is to enable a channel again if it is not explicitly disabled. So, every policy not disabling the channel would enable it. This would probably confuse users that manually disable a channel for whatever reason.

In short, I don't see a valid use-case..

**Comment by steff123** [comment]
1) Valid point if it is correct. I am not deep in the details of LND. The flag disable/enable is part of the node_policy like the fees. Hence I have thought you can only disable it one direction.

2) Confusing the user: The same would happen if one run charge-lnd and manually updating the fees
Conflict with mc: because both are core features of lnd, mc should handle it properly. Otherwise It would be a bug.

**Comment by accumulator** [comment]
I looked a bit more into this. It seems the enabled/disabled state is just for one side of the channel. Furthermore, it seems LND does distinguish between manual enable/disable requests and its internal control mechanisms, so it seems there would be a valid use-case after all :)

**Comment by accumulator** [comment]
I added a channel disable strategy in 022d8bf65a96eaae6fbb4947a521c6ede8b7ac2a

@steff123 can you test and let me know if it works as you expect?

**Comment by accumulator** [comment]
released with 0.2.4

**Comment by steff123** [comment]
thank you for the implementation. Sorry I was three weeks on holidays. Have to upgrade my node to 0.13.1 the next days. Then I am going to test it.
